packages/theming/apple-theme/src/__tests__/apple-theme.test.ts

```diff
@@ -1,9 +1,14 @@
-import { createAppleTheme } from '../createAppleTheme';
 import { createMacOSColorAliasTokens, createMacOSShadowAliasTokens } from '../createMacOSAliasTokens';
 import { getIsHighContrast, setIsHighContrast } from '../appleHighContrast.macos';
 import { AppearanceOptions } from '@fluentui-react-native/theme-types';
+import { createAppleTheme } from '../createAppleTheme';
+
+// For some reason the automatically mocked AccessibilityInfo object is the react-native version, not the
+// react-native-macos version, which doesn't contain the isHighContrastEnabled function that is accessed in
+// createAppleTheme. Use the actual module instead
+jest.dontMock('react-native-macos/Libraries/Components/AccessibilityInfo/AccessibilityInfo');
```

Comment:
Maybe make an issue and leave the number here for follow-up?

Reply:
Going to check this in as is to get the color token work unblocked for now